addons/rose/addon/components/rose/header/brand/index.hbs

```diff
@@ -7,8 +7,7 @@
   ...attributes
   class='rose-header-brand'
   title={{@text}}
-  @route={{if @route @route 'scopes.scope'}}
-  @model={{if @model @model 'global'}}
+  @route={{if @route @route 'index'}}
```

Comment:
reverted this to how it was before because it didn't fix the bug for the "Manage Projects in Org" page (it only worked for "Manage Custom Scopes" page).